usagov/usagov-2021
# USAGOV-2061: Provide the $CRON_DIGEST value to deploy-cron

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2061

## Description

Replace `$CMS_DIGEST` with `$CRON_DIGEST` in a spot that should have had `$CRON_DIGEST` all long. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

Look it over! 

If it looks good and you approve and merge, deploy the cron to dev and then observe that the cron app's docker image is specified by sha256, not by tag: 

```
cf target -s dev
cf app cron
```

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
